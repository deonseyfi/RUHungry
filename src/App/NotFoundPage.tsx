import { Button } from '@material-ui/core';
import React from 'react';
import { Link } from 'react-router-dom';

/**
 * 404 page to hit if directed to invalid path.
 *
 * @returns {React.ReactElement} Not Found Page component.
 */
const NotFoundPage = (): React.ReactElement => (
    <div style={{ textAlign: 'center' }}>
        <h1>404 NOT FOUND</h1>
        <Button
            variant='outlined'
            color='inherit'
            component={Link}
            to='/'
        >Go Home
        </Button>
    </div>
);
export default NotFoundPage;
