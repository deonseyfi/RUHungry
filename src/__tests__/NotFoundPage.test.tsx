import React from 'react';
import { mount, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { MemoryRouter } from 'react-router-dom';
import NotFoundPage from '../App/NotFoundPage';

configure({ adapter: new Adapter() });

describe('Not Found Page test', () => {
    test('renders Not Found Page', () => {
        const wrapper = mount(
            <MemoryRouter>
                <NotFoundPage />
            </MemoryRouter>,
        );
        expect(wrapper).toMatchSnapshot();
    });
});
