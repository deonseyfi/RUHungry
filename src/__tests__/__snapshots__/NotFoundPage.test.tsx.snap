// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Not Found Page test renders Not Found Page 1`] = `ReactWrapper {}`;
